wdl/FilterGenotypes.wdl

```diff
@@ -0,0 +1,325 @@
+    maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
+  }
+}
+
+task OptimizeCutoffs {
+  input {
+    File table
+    File? script
+    Float fmax_beta
+    String? args
+    String output_prefix
+    String sv_pipeline_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+                               cpu_cores: 1,
+                               mem_gb: 3.75,
+                               disk_gb: ceil(100 + size(table, "GB")),
+                               boot_disk_gb: 10,
+                               preemptible_tries: 3,
+                               max_retries: 1
+                             }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  output {
+    File out = "~{output_prefix}.tar.gz"
+    String filter_args = read_lines("~{output_prefix}/~{output_prefix}.filter_args.txt")[0]
+  }
+  command <<<
+    set -euo pipefail
+
+    mkdir ~{output_prefix}
+    python ~{default="/opt/sv-pipeline/scripts/optimize_sl_cutoffs.py" script} \
+      --table ~{table} \
+      --out-dir ~{output_prefix} \
+      --out-name ~{output_prefix} \
+      --beta ~{fmax_beta} \
+      ~{args}
+
+    tar czf ~{output_prefix}.tar.gz -C ~{output_prefix}/ .
+  >>>
+  runtime {
+    cpu: select_first([runtime_attr.cpu_cores, default_attr.cpu_cores])
+    memory: select_first([runtime_attr.mem_gb, default_attr.mem_gb]) + " GiB"
+    disks: "local-disk " + select_first([runtime_attr.disk_gb, default_attr.disk_gb]) + " HDD"
+    bootDiskSizeGb: select_first([runtime_attr.boot_disk_gb, default_attr.boot_disk_gb])
+    docker: sv_pipeline_docker
+    preemptible: select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+    maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
+  }
+}
+
+task FilterVcf {
+  input {
+    File vcf
+    File ploidy_table
+    File? script
+    String? args
+    String output_prefix
+    String sv_pipeline_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+                               cpu_cores: 1,
+                               mem_gb: 3.75,
+                               disk_gb: ceil(100 + size(vcf, "GB") * 2),
+                               boot_disk_gb: 10,
+                               preemptible_tries: 3,
+                               max_retries: 1
+                             }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  output {
+    File out = "~{output_prefix}.vcf.gz"
+    File out_index = "~{output_prefix}.vcf.gz.tbi"
+  }
+  command <<<
+    set -euo pipefail
+
+    # Convert format
+    python ~{default="/opt/sv-pipeline/scripts/apply_sl_filter.py" script} \
+      --vcf ~{vcf} \
+      --out tmp.vcf.gz \
+      --ploidy-table ~{ploidy_table} \
+      ~{args}
+
+    bcftools +fill-tags tmp.vcf.gz -- -t AC,AN,AF \
```

Comment:
In the other workflow, I wanted to be able to preserve AF when subsetting to a small fraction of the cohort (e.g. the holdout sample set in AoU). Here, I wanted to make sure that filtered genotypes are no longer counted, since presumably the new AF would be more accurate.